corehq/apps/hqcase/management/commands/ptop_fast_reindexer.py

```diff
@@ -34,7 +34,12 @@ def view_starting(self, db, view_name, kwargs, total_emitted):
             total_emitted,
             total_emitted + kwargs['limit'] - 1)
         )
-        self.log('  startkey={}, startkey_docid={!r}'.format(kwargs.get('startkey'), kwargs.get('startkey_docid')))
+        startkey = kwargs.get('startkey')
+        if isinstance(startkey, basestring):
+            startkey = startkey.encode('utf8')
+        elif isinstance(startkey, list):
+            startkey = [i.encode('utf8') for i in startkey if isinstance(i, basestring)]
+        self.log('  startkey={}, startkey_docid={!r}'.format(startkey, kwargs.get('startkey_docid')))
```

Comment:
Would `startkey={!r}` work? That would make it unnecessary to do the encoding above.

Another possibility would be to make the template a unicode string: `u'  startkey={}, startkey_docid={!r}'`